events/importer/helper/importers/vapaaehtoistyofi/record.py

```diff
@@ -0,0 +1,135 @@
+import datetime
+
+
+class Record:
+    STATUS_PUBLISHED = 1
+    STATUS_DRAFT = 0
+    STATUS = [STATUS_PUBLISHED, STATUS_DRAFT]
+
+    LOCALE_FI = 'fi'
+    LOCALE_SE = 'se'
+    LOCALE_EN = 'en'
+    LOCALE = [LOCALE_FI, LOCALE_SE, LOCALE_EN]
+
+    TASK_URL_FORMAT = 'https://vapaaehtoistyo.fi/%s/task/%s'
+
+    def __init__(self, json_dict):
+        if json_dict:
```

Comment:
I find it better to destruct the json before initializing a new class instance or at least use another constructor method like `from_json(json_object)`. So I would map the json values to the class instance with a json serializer. Then the class instance creation would be as simple as `Record(**record_dict)`.